openjdk8/pom.xml

```diff
@@ -90,6 +97,30 @@
 	  </execution>
         </executions>
       </plugin>
+      <plugin>
+	<groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>1.5.0</version>
+        <executions>
+          <execution>
+            <id>structure-test</id>
+            <phase>verify</phase>
+            <goals>
+              <goal>exec</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <executable>../../scripts/ext_run.sh</executable>
+          <workingDirectory>${project.build.directory}</workingDirectory>
```

Comment:
Really the only reason for doing this is that the structure test framework doesn't clean up temporary files if there is a test failure. This way, temp files won't pollute the source directories.  Also, note that ../../scripts/ext_run.sh is the relative path from the `project.build.directory`